### PR TITLE
adds syntax exercise in basic exercises

### PR DESCRIPTION
**Pull request by Felienne**

### CODE REVIEW

File: exercises.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ The goal of this exercise is to practice to get a first impression of code and t
 
 #### (1 min) Independently - Glance at the code
 
-It's important that is an immediate reaction.
+It's important that what you use is your  immediate reaction, don't overthink it!
```
Comment (katjam) [inline exercises.md:22]: Great!

At some point we should sync this doc with my runner slide data - I could probably use the markdown from here quite easily to populate https://github.com/CodeReadingClubs/runner/blob/main/src/Exercises.elm

File: exercises.md [hunk 2]
```diff
@@ -37,6 +37,27 @@ Reflect also on what kind of knowledge you used in this exercise:
...
 
+### Syntax knowledge
+
+The goal of this exercise is to practice to make sure everyone in club is familiar with syntactic elements of the code. 
+
+#### (5 min) Independently - Examine syntax
+
+Look at the code and examine syntactic elements. Do you know the meaning of all elements? You can use these questions as a guide:
+
+* Is it clear to you what the role of each block in the code is (function, condition, repetition etc)?
```
Comment (katjam) [inline exercises.md:48]: is role the right word here?

Comment (Felienne) [reply exercises.md:48]: I thought so, but I am not a native speaker :) What I mean to say is: do you know if each block is a function or a condition or a repetition etc. Feel free to rewrite!

File: exercises.md (hunk 1 above)
Comment (Felienne) [reply exercises.md:22]: can we do that automatically? Would be cool!

Comment (katjam) [reply exercises.md:22]: Yeah - would be good. I don't think it'd be too tricky.

File: exercises.md (hunk 2 above)
Comment (katjam) [reply exercises.md:48]: I'll leave it - it mostly works... but there might be a better work, 'function' is not a good one in this case, but probably more what you mean.

**Pull request merged**